website/docs/compaction.md

```diff
@@ -5,52 +5,156 @@ toc: true
+
+- `UnBoundedCompactionStrategy`: This strategy compacts all base files with its respective log files. For example, if you use a schedule strategy, like `num_commit=5`, all file groups with updates between those 5 commits will be prepped for compaction. There are no further trimming or settings you need to take into consideration. 
+
+The additional configurations based on the strategy values are below:
+
+- `hoodie.compaction.target.io`: This is the number of MBs to spend during the compaction run for the `LogFileSizeBasedCompactionStrategy`, `LogFileNumBasedCompactionStrategy` and `BoundedIOCompactionStrategy`. This value helps bound ingestion latency while compaction runs. 
+
+#### Example of how a compaction plan and strategy work in Hudi: 
+
+Let’s say you set the schedule and compaction strategy to these values: 
+
+```
+hoodie.compact.inline.trigger.strategy = num_commits 
+hoodie.compact.inline.max.delta.commits = 5
+hoodie.compaction.strategy = org.apache.hudi.table.action.compact.strategy.LogFileSizeBasedCompactionStrategy
+hoodie.compaction.logfile.size.threshold=104857600
+hoodie.compaction.target.io = 10240
+```
+
+After reaching 5 commits, Hudi evaluates the entire table to identify file groups with log files that meet or exceed the 100MB file size threshold. Let’s assume that out of 100 file groups, 50 file groups have log files that meet or exceed this threshold. Among these 50 file groups, Hudi accumulates log files up to 10GB, which is the specified target threshold. Once the 10GB log files limit is reached, Hudi finalizes the compaction plan and creates a `.compaction.requested` instant in the `.hoodie` directory. This results in a pending compaction. From there, the compaction plan is executed. 
+
+## Compaction deployment models:
+Once you have configured your compaction strategy, you need to execute it. Below are several options for how you can do this:
+
+### Inline Compaction
+Inline compaction refers to the process of executing the compaction process as part of the data ingestion pipeline, rather than running them asynchronously as a separate job. With inline compaction, Hudi will schedule, plan and execute the compaction operations after each commit is completed. This is the simplest deployment model to run because it’s easier to manage than running different asynchronus Spark jobs (see below). This mode is only supported on Spark Datasource, Spark-SQL and DeltaStreamer in sync-once mode. In the next section, we’ll go over code snippets you can use to get started with inline clustering. To run inline compaction with DeltaStreamer in continuous mode, you must pass the flag `--disable-compaction` so the async compaction is disabled (see below). With inline compaction, your data latency could be higher because you'll block any writer from ingesting data when compaction is being executed.
```

Comment:
to get started with inline clustering -> compaction?

Reply:
fixed